src/main/java/ttt/GameType.java

```diff
@@ -1,17 +1,18 @@
 package ttt;
 
 public enum GameType {
-    HUMAN_VS_HUMAN(1, "Human vs Human"),
-    HUMAN_VS_UNBEATABLE(2, "Human vs Unbeatable"),
-    UNBEATABLE_VS_HUMAN(3, "Unbeatable vs Human");
-
+    HUMAN_VS_HUMAN("Human vs Human", 1, 10),
+    HUMAN_VS_UNBEATABLE("Human vs Unbeatable", 2, 4),
+    UNBEATABLE_VS_HUMAN("Unbeatable vs Human", 3, 4);
 
     private final int gameType;
+    private final int maximumDimension;
     private final String gameName;
 
-    GameType(int gameType, String gameName) {
-        this.gameType = gameType;
+    GameType(String gameName, int gameType, int maximumDimension) {
         this.gameName = gameName;
+        this.gameType = gameType;
```

Comment:
The `int` is what the user enters on the command line, thus the display on the commandLine is 
`Enter 1 for human vs human
Enter2 for human vs computer... `

So in the Prompt I can loop around the values in the array and get the game type (int value) and game name (string value)